mms/mxnet_model_server.py

```diff
@@ -156,15 +156,15 @@ def _arg_process(self):
             service_file = os.path.join(models[0][2], manifest['Model']['Service'])
 
             class_defs = self.serving_frontend.register_module(self.args.service or service_file)
-            
-            if len(class_defs) < 1:
-                raise Exception('User defined module must derive base ModelService.')
-            # The overrided class is the last one in class_defs
-            mode_class_name = class_defs[-1].__name__
+            class_defs = filter(lambda c: len(c.__subclasses__()) == 0, class_defs)
+
+            if len(class_defs) != 1:
+                raise Exception('There should be one user defined service derived from ModelService.')
```

Comment:
Can't there be more than one?

Reply:
I think one service class make senses, if it is more than 1, we will not know which one to start the server.